src/main/scala/karazin/scala/users/group/week1/homework/Homework.scala

```diff
@@ -40,28 +40,71 @@ object Homework :
 
   object `Boolean Operators` :
 
-    val int = 42
+    def not(b: Boolean): Boolean = {
+      if b then false
+      else true
+    } // here is my greatest solution
 
-    def not(b: Boolean): Boolean = ??? // here is my greatest solution
+    def and(left: Boolean, right: Boolean): Boolean = {
```

Comment:
`and` must be eager, meaning that if `right` is not used then it should not be evaluated.